app/api/events.py

```diff
@@ -210,15 +278,33 @@ def before_get_object(self, view_kwargs):
                 else:
                     view_kwargs['id'] = None
 
-    decorators = (is_coorganizer, )
+    def after_get(self, result):
+        result = changeToTimezone(result)
```

Comment:
@LuD1161 after changing event_url to fields.Str(), get post is working fine on nextgen branch, Here is the post data https://pastebin.com/4XP4M8G6 and here is the response https://pastebin.com/zFjCntHi and